provider/inspec.rb

```diff
@@ -267,8 +267,22 @@ def sub_property_descriptions(property)
       end
     end
 
-    def markdown_format(property)
-      "    * `#{property.out_name}`: #{property.description.split("\n").join(' ')}"
+    # Recursively calls itself on any arrays or nested objects within this property, indenting
+    # further for each call
+    def markdown_format(property, indent = 1)
```

Comment:
It feels a little silly to start iterating in `sub_property_descriptions` and then continue iteration here, but I guess we want to act a little bit like a `do while` so it makes enough sense.